Behat/Context/UIContext.php

```diff
@@ -206,4 +206,50 @@ public function iClickOnTheLabel($label) {
     $this->iClickOnTheElementWithXPath($xpath);
   }
 
+  /**
+   * Get element by xpath.
+   */
+  protected function getElementByXpath($xpath_element) {
+    $page = $this->getSession()->getPage();
+    $findelement = $page->find('xpath', $xpath_element);
+    return $findelement;
+  }
+
+  /**
+   * Check if a type of element has an attribute with an specific value.
+   *
+   * @Then the :element element of :type type should have the :attribute attribute with :value value
+   */
+  public function theElementShouldHaveAttributeWithValue($element, $type, $attribute, $value, $not = NULL) {
```

Comment:
Use function initialization with TRUE:
`public function theElementShouldHaveAttributeWithValue($element, $type, $attribute, $value, $not = FALSE) {`